Modules/Core/QuadEdgeMesh/test/itkQuadEdgeMeshEulerOperatorDeleteCenterVertexTest.cxx

```diff
@@ -82,13 +83,8 @@ itkQuadEdgeMeshEulerOperatorDeleteCenterVertexTest(int argc, char * argv[])
   }
   std::cout << "OK" << std::endl;
 
-  const std::string className = deleteCenterVertex->GetNameOfClass();
-  const std::string requiredClassName{ "QuadEdgeMeshEulerOperatorDeleteCenterVertexFunction" };
-  if (className != requiredClassName)
-  {
-    std::cout << className << " != " << requiredClassName << std::endl;
-    return EXIT_FAILURE;
-  }
+  ITK_TEST_EXPECT_EQUAL((const std::string) "QuadEdgeMeshEulerOperatorDeleteCenterVertexFunction",
+                        (const std::string)deleteCenterVertex->GetNameOfClass());
```

Comment:
Thanks for addressing this test code redundancy, Jon! I'm sorry I'm too late review, as I see that your pull request is already merged 🤷 I would have suggested not to use C-style casts, like `(const std::string)`. They might cause warnings from static analysis tools, like Clang-Tidy:

> C-style casts are discouraged; use static_cast [google-readability-casting]

And [Resharper](https://www.jetbrains.com/help/resharper/Reference__Code_Inspections_CPP.html#ClangTidy):

> C-style cast is used instead of a C++ cast  

Actually I think these would be nice candidates for C++17 `std::string_view`. For example:

    ITK_TEST_EXPECT_EQUAL(std::string_view("QuadEdgeMeshEulerOperatorDeleteCenterVertexFunction"),
                          std::string_view(deleteCenterVertex->GetNameOfClass()));

Shall I make that a new pull request?

Reply:
Please go ahead.

Reply:
- Just for the record, my new pull request is at https://github.com/InsightSoftwareConsortium/ITK/pull/4587